pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -245,6 +245,19 @@ protected Policies getNamespacePolicies(String property, String cluster, String
         }
     }
 
+    protected LocalPolicies getNamespaceLocalPolicies(String property, String cluster, String namespace) {
```

Comment:
Ok. But I think we introduced local-policies to keep cluster-level policy information such as `BundleData` and it keeps global-policies clean.